components/reducers/entityReducer.js

```diff
@@ -0,0 +1,122 @@
+export const initialState = {
+  values: [],
+  attributes: [],
+  columns: [],
+  entity_type_name: null,
+  isLoading: false,
+  isRefresh: true,
+  isSaving: false,
+  isDeleting: false,
+  show: false,
+  entity_type_id: null,
+  modalContent: null,
+};
+
+const LOADING = "LOADING";
```

Comment:
Kindly refactor this to a separate file. You can add it to reducers > actions.js